browser/extensions/brave_extension_provider_browsertest.cc

```diff
@@ -65,6 +66,15 @@ IN_PROC_BROWSER_TEST_F(BraveExtensionProviderTest, PDFJSInstalls) {
       "window.domAutomationController.send("
         "!!document.body.querySelector(\"#chrome-pdfjs-logo-bg\"))", &pdfjs_exists));
   ASSERT_TRUE(pdfjs_exists);
+
+  // Ensure that the extension prefix is not in the display URL (brave-browser#368)
+  EXPECT_STREQ(contents->GetController().GetLastCommittedEntry()
+               ->GetVirtualURL().spec().c_str(),
+               url.spec().c_str()); // no "chrome-extension://" prefix
+  std::string internal_url =
+    "chrome-extension://oemmndcbldboiebfnladdacbdfmadadm/" + url.spec();
```

Comment:
nit: continuation of a line should have 4 spaces (or more if some kind of alignment with next/previous line helps readability)